automata/pda/configuration.py

```diff
@@ -27,3 +28,16 @@ def __repr__(self) -> str:
         return "{}({!r}, {!r}, {!r})".format(
             self.__class__.__name__, self.state, self.remaining_input, self.stack
         )
+
+    def __eq__(self, other: Any) -> bool:
+        """Return True if two PDAConfiguration are equivalent"""
+        if not isinstance(other, PDAConfiguration):
+            return NotImplemented
+
+        if (
```

Comment:
```suggestion
        return (
```
No need to return true and false in separate statements.